### PR TITLE
Add UI testing for TGIS Standalone

### PR DESCRIPTION
**Pull request by bdattoma**
Adding 2 new e2e UI tests for TGIS Standalone serving runtime: with and without GPU node

### CODE REVIEW

**Comment by bdattoma** [comment]
PR validation:
- TGIS standalone UI test: `rhods-ci-pr-test/2377/`
- TGIS standalone UI test on GPU:  `rhods-ci-pr-test/2378/`

**Review by jgarciao — APPROVED** [review]

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_other_runtimes_UI.robot [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    ...    User Can Fetch Number Of Requests Over Defined Time
+    ...    thanos_url=${THANOS_URL}    thanos_token=${THANOS_TOKEN}
+    ...    model_name=${model_name}    query_kind=single    namespace=${test_namespace}    period=5m    exp_value=1
+    Delete Model Via UI    ${model_name}
+
+Verify Model Can Be Served And Query On A GPU Node Using The UI  # robocop: disable
+    [Documentation]    Basic tests for preparing, deploying and querying a LLM model on GPU node
+    ...                using Single-model platform and TGIS Standalone runtime.
+    [Tags]    Sanity    Tier1    ODS-2612   Resources-GPU
+    [Setup]    Run    git clone https://github.com/IBM/text-generation-inference/
```
Comment (manosnoam) [inline ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_other_runtimes_UI.robot:54]: another clone of `text-generation-inference`...

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_other_runtimes_UI.robot [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+${TGIS_RUNTIME_NAME}=    tgis-runtime
+@{SEARCH_METRICS}=    tgi_    istio_
+
+
+*** Test Cases ***
+Verify Non Admin Can Serve And Query A Model Using The UI  # robocop: disable
+    [Documentation]    Basic tests leveraging on a non-admin user for preparing, deploying and querying a LLM model
+    ...                using Single-model platform and TGIS Standalone runtime.
+    [Tags]    Sanity    Tier1    ODS-2611
+    [Setup]    Run    git clone https://github.com/IBM/text-generation-inference/
```
[the remark this answers is not in the document]
Comment (bdattoma) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_other_runtimes_UI.robot:26]: the clone will fail but won't fail the test

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_other_runtimes_UI.robot (hunk 1 above)
Comment (bdattoma) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_other_runtimes_UI.robot:54]: yes. The idea around this suite is to put tests about more basic tests for serving runtimes which will come. Not all of them will need the same repos. So I clone it again here

**Review by FedeAlonso — APPROVED** [review]

**Pull request merged**